test-e2e/fixtures/config-test-fixtures.ts

```diff
@@ -13,13 +17,20 @@ const baseURLProtocol = async ({ baseURL }, use): Promise<void> => {
 
 export const test = base.extend<{ rootDomain: string, baseURL: string, protocol: string }>({
   rootDomain: [rootDomain, { scope: 'test' }],
-  protocol: [baseURLProtocol, { scope: 'test' }]
+  protocol: [baseURLProtocol, { scope: 'test' }],
+  page: async ({ page }, use) => {
+    if (isNoServiceWorkerProject(test)) {
+      test.skip()
+      return
+    }
+    await use(page)
+  }
 })
 
 /**
  * You should use this fixture instead of the `test` fixture from `@playwright/test` when testing path routing via the service worker.
  */
-export const testPathRouting = base.extend<{ rootDomain: string, baseURL: string, protocol: string }>({
+export const testPathRouting = test.extend<{ rootDomain: string, baseURL: string, protocol: string }>({
```

Comment:
idk why this wasn't being done before, but all of our custom test fixtures now extend from our `test` fixture above, so we get the `isNoServiceWorkerProject` check for free without modifying all the others.